client/my-sites/marketing/tools/index.tsx

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import page from 'page';
+import PropTypes from 'prop-types';
+import React, { Fragment, FunctionComponent } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionCard from 'components/action-card';
+import Button from 'components/button';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import MarketingToolFeature from './feature';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+interface MarketingToolsProps {
+	selectedSiteSlug: string | null;
+}
+
+export const MarketingTools: FunctionComponent< MarketingToolsProps > = ( {
+	selectedSiteSlug,
+} ) => {
+	const translate = useTranslate();
+
+	const handleBoostMyTrafficClick = () => {
+		page( selectedSiteSlug ? `/marketing/traffic/${ selectedSiteSlug }` : '/marketing/traffic' );
+	};
+
+	const handleStartSharingClick = () => {
+		page(
+			selectedSiteSlug
+				? `/marketing/sharing-buttons/${ selectedSiteSlug }`
+				: '/marketing/sharing-buttons'
+		);
```

Comment:
What about using this shorter approach for those redirects?

```javascript
page( `/marketing/sharing-buttons/${ selectedSiteSlug || '' }` );
```

Reply:
I am always about making things simpler!